### DISCUSSION TITLE
2007/04/30/vi-%E4%B8%AD%E7%9A%84%E6%95%B0%E6%8D%AE%E7%A9%BA%E9%97%B4

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#157] [Announcements]
# 2007/04/30/vi-%E4%B8%AD%E7%9A%84%E6%95%B0%E6%8D%AE%E7%A9%BA%E9%97%B4

LabVIEW 由于比其它语言采用了更多的值传递方式，这必然会影响它的运行效率，也使得 LabVIEW 在这方面要采取一些其它语言不需要的应对措施，尽量提高效率。优化之一是子 VI 中局部变量使用的内存的分配方式。

https://blog.qizhen.xyz/2007/04/30/vi-%E4%B8%AD%E7%9A%84%E6%95%B0%E6%8D%AE%E7%A9%BA%E9%97%B4

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Unknown
2007年06月11日 11:58 编辑
您好
请教一下用labview怎么设计系统动态特性补偿仪，频率分析，相位差设计这三个功能。
毕业论文中需要，最好是文字型资料。恳求援助。谢谢。

回复
杰
2007年05月10日 19:46 编辑
鬼宿，你可以使用ring控件实现你需要的for循环调用。
GPIB有例子的，自己多看看。
类似的问题，多上一些LV的论坛看看，这样问小阮，他会累趴下的，呵呵。
来这里看看，加深一些对LV的理解。毕竟现在LV的书很少。
不是很合适咨询一些基础的问题，呵呵。
话有点糙，见谅。

回复
Karen-momo
2007年05月10日 19:17 编辑
哇~~~好深奥

回复
秋
2007年05月8日 15:21 编辑
虽然看不懂你这是个啥程序~但我还是得来看看你~

回复
磊
2007年05月5日 15:14 编辑
我是labview的初学者，有2个问题想请教一下，一个就是我想用for循环调用字符串，这个怎么调用，每次循环都要调用不同的字符串（字符串已经提前输入，人不能在运行中随意更改）。还有就是我不太会用GPIB发送和接受数据，能指导指导我么？？

回复
arrowpig
2007年05月1日 08:47 编辑
还没有写完吧，五一后再来拜读.

回复
